core/src/assets/defaultConfig.json

```diff
@@ -60,6 +60,28 @@
           ]
         }
       },
+      "DISABLED_NODES": {
+        "isEnabled": true,
+        "config": []
+      },
+      "HIDDEN_NAMESPACES": {
+        "isEnabled": true,
+        "config": [
```

Comment:
Everywhere `config` is an object, I believe we should stick to that approach, in case we would want to add something else to this configuration in the future.